framework/src/main/resources/config.conf

```diff
@@ -634,34 +645,33 @@ genesis.block = {
     }
   ]
 
-  timestamp = "0" #2017-8-26 12:00:00
+  timestamp = "0" # Genesis block timestamp, milli seconds
 
   parentHash = "0xe58f33f9baf9305dc6f82b9f1934ea8f0ade2defb951258d50167028c780351f"
```

Comment:
Is there a comment about how the parentHash is calculated?

Reply:
As far as I understand, it's a random value.